packages/terra-list/src/SelectableUtils.js

```diff
@@ -165,17 +237,32 @@ const newPropsForItem = (item, index, onClick, onKeyDown, hasChevrons, selectedI
   return newProps;
```

Comment:
Does `newPropsForItem` merge custom props into the newProps?

Reply:
Yes, This happens when you create a element with React.cloneElement. [Github](https://github.com/cerner/terra-core/blob/List-Static-Section-Headers/packages/terra-list/src/SelectableList.jsx#L77)